css/main.css

```diff
@@ -368,7 +368,7 @@ footer {
   justify-content: center;
   letter-spacing: 0.03em;
   color: #fff;
-  background-color: orange;
+  background-color:  #F55800;
```

Comment:
- [ ] You've done a great job fixing the color of the button :clap: :clap:, you still only have to add the interactions styles (hover and active) to nail it :heart_eyes: implementing them is required :sparkles: 
Here's a screenshot of the button interactions from the Figma template :point_down:
![Screenshot from 2022-05-20 11-12-08](https://user-images.githubusercontent.com/87965534/169523984-06be5570-7183-4384-9de9-83b9e3688778.png)